src/OrbitGl/BatchRenderGroupTest.cpp

```diff
@@ -11,6 +11,12 @@
 #include "OrbitGl/BatchRenderGroup.h"
 
 namespace orbit_gl {
+
+bool operator==(const StencilConfig& lhs, const StencilConfig& rhs) {
+  return std::tie(lhs.enabled, lhs.pos[0], lhs.pos[1], lhs.size[0], lhs.size[1]) ==
+         std::tie(rhs.enabled, rhs.pos[0], rhs.pos[1], rhs.size[0], rhs.size[1]);
+}
```

Comment:
As far as I can see you define this operator only in the cpp file, but without a declaration in the header. This can easily lead to ODR violations if someone else also decides to add a comparison operator.

I see 2 ways to resolve this:

1. Also declare it in the header (as a hidden friend as usual). In that case you should also define `operator!=` for consistency.
2. You make this an implementation detail of the unit tests. That means putting it into an anonymous namespace and rename it to some generic function name. Keeping this as an operator would be confusing. We also don't do this anywhere else.

(I would prefer option 1)

Reply:
Implemented option 1